model/gen-ai/registry.yaml

```
@@ -253,6 +255,93 @@ groups:
             Client-side operations are actions taken on the user's end or within the client application.
           Datastore: A tool used by the agent to access and query structured or unstructured external data for retrieval-augmented tasks or knowledge updates.
         examples: ['function', 'extension', 'datastore']
+
+      - id: gen_ai.tool.call.arguments
+        stability: development
+        type: any
+        brief: Parameters passed to the tool call.
+        note: >
+          > [!WARNING]
+          > This attribute may contain sensitive information.
+
+          It's expected to be an object - in case a serialized string is available
+          to the instrumentation, the instrumentation SHOULD do the best effort to
+          deserialize it to an object.
+        examples: 
+        - |
+            [
+              {
+                "location": "San Francisco?",
+                "date": "2025-10-01"
+              }
+            ]
+        
+      - id: gen_ai.tool.call.result
+        stability: development
+        type: any
+        brief: The result returned by the tool call (if any and if execution was successful).
+        note: |
+          > [!WARNING]
+          > This attribute may contain sensitive information.
+
+          It's expected to be an object - in case a serialized string is available
+          to the instrumentation, the instrumentation SHOULD do the best effort to
+          deserialize it to an object.
+
+        examples: 
+        - |
+            {
+              "temperature_range": {
+                "high": 75,
+                "low": 60
+              },
+              "conditions": "sunny"
+            }
+        
+      - id: gen_ai.tool_definitions.raw
```

```suggestion
      - id: gen_ai.tool_definitions
```

it seems some changes got lost

---

@lmolkova : repushed the changes again, its visible in latest commit. kindly review.